scene/gui/line_edit.cpp

```diff
@@ -270,14 +270,14 @@ void LineEdit::gui_input(const Ref<InputEvent> &p_event) {
 						caret_column = selection.begin;
 					} else if (b->is_double_click()) {
 						// Double-click select word.
+						selection.last_dblclk = OS::get_singleton()->get_ticks_msec();
```

Comment:
Not sure about moving this out of the loop, if there is no text and you triple click we will create a section of length `0`, which might cause some issues. Either we put this back in the loop, or triple click needs to check the text length.

Reply:
OK, looking at 3.x code and seems that there was no check of that sort.

But you're right, it may be be safer to ensure that text length is > 0 in the triple click block.